lib/include/openamp/virtio.h

```diff
@@ -68,6 +68,7 @@ struct virtio_device_id {
 #define VIRTIO_TRANSPORT_F_END        32
 
 typedef struct _virtio_dispatch_ virtio_dispatch;
+typedef void (*vdev_reset_cb)(struct virtio_device *vdev);
```

Comment:
when is this suppose to be used or called?

Where are we holding on to it?

Reply:
this is used by the virtio host side application to get notified when the guest side got reset